src/es6/Entity.js

```diff
@@ -37,6 +38,9 @@ export class Entity {
 
 		return this._entity.getLinkByRel('self').href;
 	}
+	async subEntitiesLoaded() {
```

Comment:
This could just be a function that returns a promise...

Reply:
Yup!